src/index.js

```diff
@@ -345,7 +345,7 @@ class Node extends EventEmitter {
       let t
 
       if (typeof Transport === 'function') {
-        t = new Transport()
+        t = new Transport({libp2p: this})
```

Comment:
Linting fix:
```suggestion
        t = new Transport({ libp2p: this })
```

Reply:
Other than the linting issue this looks fine. Travis is failing due to this and commitlint. I went ahead and disabled commitlint on master so it doesn't block tests going forward. If you rebase master and fix the linting issue I suggested above, travis tests should properly run.

I'll update the commit message on squash/merge.